cli/cli/cli.go

```diff
@@ -131,15 +132,11 @@ func (curveadm *CurveAdm) init() error {
 	configure.ReplaceGlobals(config)
 
 	// (3) Init logger
-	now := time.Now().Format("2006-01-02_15-04-05")
-	logpath := fmt.Sprintf("%s/curveadm-%s.log", curveadm.logDir, now)
-	if err := log.Init(config.GetLogLevel(), logpath); err != nil {
-		return errno.ERR_INIT_LOGGER_FAILED.E(err)
-	} else {
-		log.Info("Init logger success",
-			log.Field("LogPath", logpath),
-			log.Field("LogLevel", config.GetLogLevel()))
-	}
+	logpath := fmt.Sprintf("%s/curveadm.log", curveadm.logDir)
```

Comment:
Why is the date-related information removed from the log name?

Reply:
The content of the same file reaches the specified number to be compressed. Passing the date will result in too many small files and cannot be cleared